server/src/lib/model/db/import-sentences.ts

```diff
@@ -109,13 +114,34 @@ export async function importSentences(pool: any) {
   await pool.query(
     'DELETE FROM sentences WHERE id NOT IN (SELECT original_sentence_id FROM clips)'
   );
+
   await pool.query('UPDATE sentences SET is_used = FALSE');
 
+  const [rows] = await pool.query(
+    'SELECT bucket, COUNT(bucket) AS count FROM sentences GROUP BY bucket'
```

Comment:
In the name of DRY, this query string should probably be abstracted somewhere, since we use it in db.getClipBucketCounts